### PR TITLE
Pygtftk

### PR DESCRIPTION
**Pull request by guillaumecharbonnier**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This recipe is adding [pygtftk](https://github.com/dputhier/pygtftk).

@bioconda/core
I had to add this line ```ln -s $CC `dirname $CC`/gcc``` into the build.sh ([details](https://github.com/dputhier/pygtftk/issues/21)) to successfully compile but it does not look like something other packages need so maybe there is a more elegant way to do with the setup.py.

### CODE REVIEW

File: recipes/pygtftk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - pandas >=0.23.3
+    - scipy >=1.1.0
+    - pybedtools >=0.7.8
+    - pybigwig >=0.3
+    - pyyaml >=3.12
+    - requests >=2.13.0
+    - cffi >=1.10.0
+    - pyparsing >=2.2.0
+    - biopython >=1.69
+    - zlib
```
Comment (bgruening) [inline recipes/pygtftk/meta.yaml]: zlib can be removed here as it is a run_export nowadays.

File: recipes/pygtftk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  sha1: 23862a12605a02de001d540c4e2fa5688294d662
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.6,<3.7
```
Comment (bgruening) [inline recipes/pygtftk/meta.yaml]: can you `skip: True` builds instead of this pinning here

File: recipes/pygtftk/build.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ln -s $CC `dirname $CC`/gcc
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/pygtftk/build.sh]: pip install is recommend instead of setup.py usage

**Comment by bgruening** [comment]
Thanks a lot!

File: recipes/pygtftk/build.sh (hunk 3 above)
Comment (guillaumecharbonnier) [reply recipes/pygtftk/build.sh]: I have tried using `script: python -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .` in the meta.yaml but it fails for the same reason explained in my first message.
It works if I replace `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`  by `$PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .` in build.sh, but I guess this is not the recommended way.

File: recipes/pygtftk/meta.yaml (hunk 2 above)
Comment (guillaumecharbonnier) [reply recipes/pygtftk/meta.yaml]: If I understand well, I need `skip: true # [not py35 and not py36]`, keep `- python` in host but still `- python >=3.5,<3.7` in run. Am I right? 
By the way, py35 is also ok hence the difference with my commit.

**Pull request merged**